concepts/arrays-and-lists/about.md

```diff
@@ -56,7 +56,7 @@ scalar @names; # 3
 scalar ('Alice', 'Bob', 'Charlie'); # 'Charlie' (and some warnings).
 ```
 
-Read more in the folowing FAQ: [What is the difference between a list and an array?](perlfaq4listarray)
+Read more in the following article: [What is the difference between a list and an array?][perlfaq4listarray]
```

Comment:
We  could still call it a FAQ since that is what it links to.

Reply:
Indeed, we **should** use the word FAQ, since it not only describes the link, it also describes where to find it locally: `perldoc perlfaq4`.   It is what it is rather than an article.

Reply:
LGTM. I just wanted to fix the spelling and dead link.